dags/vds_pull.py

```diff
@@ -0,0 +1,131 @@
+from os import path
+import sys
+from airflow import DAG
+from datetime import datetime, timedelta
+from airflow.operators.python import PythonOperator
+from airflow.hooks.base import BaseHook
+from airflow.providers.slack.operators.slack_webhook import SlackWebhookOperator
+from airflow.providers.postgres.hooks.postgres import PostgresHook
+from airflow.providers.postgres.operators.postgres import PostgresOperator
+from airflow.models import Variable
+
+#CONNECT TO ITS_CENTRAL
+itsc_bot = PostgresHook("itsc_postgres")
+
+#CONNECT TO BIGDATA
+vds_bot = PostgresHook("vds_bot")
+
+try:
+    repo_path = '/home/gwolofs/bdit_data-sources'
+    sys.path.insert(0,path.join(repo_path,'volumes/vds/py'))
+    from vds_functions import pull_raw_vdsdata, pull_raw_vdsvehicledata, pull_detector_inventory, pull_entity_locations
+except:
+    raise ImportError("Cannot import functions from volumes/vds/py/vds_functions.py.")
+
+dag_name = 'vds_pull'
+
+# Get slack member ids
+#dag_owners = Variable.get('dag_owners', deserialize_json=True)
+#names = dag_owners.get(dag_name, ['Unknown']) #find dag owners w/default = Unknown    
+names = ['gabe']
```

Comment:
Don't forget to update this before the final merge. I updated the Airflow `dag_owners` variable with `"vds_pull": ["gabe"]`